src/main/java/core/basesyntax/AreaFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaFigure {
+    double getAreaFigure();
+}
```

Comment:
```suggestion
public interface AreaCalculator {
    double getArea();
}
```

lets remove word "Figure" from naming as we can use this method not only for Figures (some Fields or Flats could also implement AreaCalculator interface)